packages/@react-spectrum/autocomplete/src/MobileSearchAutocomplete.tsx

```diff
@@ -93,11 +107,34 @@ function _MobileSearchAutocomplete<T extends object>(props: SpectrumSearchAutoco
     }
   };
 
+  let inputProps: InputHTMLAttributes<HTMLInputElement> = {
+    type: 'hidden',
+    name,
+    value: state.inputValue
+  };
+
+  if (validationBehavior === 'native') {
+    // Use a hidden <input type="text"> rather than <input type="hidden">
+    // so that an empty value blocks HTML form submission when the field is required.
+    inputProps.type = 'text';
+    inputProps.hidden = true;
```

Comment:
false?

we'll need to make sure that these don't interfere in layout if they are no longer 'hidden', might be good to run chromatic on this PR